app/models/container_label_tag_mapping.rb

```diff
@@ -23,7 +23,7 @@ class ContainerLabelTagMapping < ApplicationRecord
 
   require_nested :Mapper
 
-  TAG_PREFIXES = ['/managed/amazon:', '/managed/kubernetes:'].freeze
+  TAG_PREFIXES = %w(_all_ amazon azure kubernetes).map { |name| "/managed/#{name}:" }.freeze
```

Comment:
why is the `_all_` here?

Reply:
Are `/managed/_all_:...` categories created anywhere?

It's a good idea to reserve a non-provider-specific prefix, currently mappings for "foo" label for all types get a `/managed/kubernetes::foo` category (`ALL_PREFIX` in UI — https://github.com/ManageIQ/manageiq-ui-classic/pull/3697/files#diff-95b50e997ace14a4e385c9a176eea25fR31).
When you actually change it, you _might_ need a migration for existing categories, and tags under them?

Also, remember these share a namespace with regular user-created categories. `_all_` is too nondescriptive IMHO; perhaps something like `mapped`?